### DISCUSSION TITLE
[firebase_app_check] Is it possible to obtain the device integrity verdict?

### DISCUSSION THREAD

**Discussion by chagaj** [firebase/flutterfire#10325] [Q&A]
Hello,

I would like to use google play integrity to obtain the integrity verdict on android devices as mentioned here - https://developer.android.com/google/play/integrity/verdict.

For example, whether a device:
"MEETS_BASIC_INTEGRITY"
"MEETS_STRONG_INTEGRITY"
"MEETS_VIRTUAL_INTEGRITY"

I am unable to find any reliable resources suggesting how to achieve this in a flutter app. Is this possible to do using the `firebase_app_check` package?

This has been mentioned here, but it is not clear if it has been implemented - https://github.com/firebase/flutterfire/pull/9646#issuecomment-1311918193

Thanks in advance.

**Comment by Gidzkid** [comment]
I would like to obtain the integrity verdict on android devices as well, is there any solution to get it?

**Comment by ChanakaWeerasinghe** [comment]
![Screenshot 2024-12-19 at 9 19 54 AM](https://github.com/user-attachments/assets/7c3eae0e-edab-47e0-a450-48dffd6c3ce3)
